### PR TITLE
Lesson 10

### PR DESCRIPTION
**Pull request by umavenki**
Lesson-10 Homework

### CODE REVIEW

**Review by toma63 — CHANGES_REQUESTED** [review]
Very nice.  There are a couple of things missing from the lesson.

You have a qualifications section instead of an experience section.  Since it's requited for the lesson, you should change it to experience.  It can be any type of experience and could even include the batchelor's in engineering.

The lesson calls for styling of the connect links and padding between sections.

**Comment by umavenki** [comment]
> Very nice. There are a couple of things missing from the lesson.
> 
> You have a qualifications section instead of an experience section. Since it's requited for the lesson, you should change it to experience. It can be any type of experience and could even include the batchelor's in engineering.
> 
> The lesson calls for styling of the connect links and padding between sections.

Hi Tom, I have changed the code as per your comments. Please review and let me know.  Thanks.

File: index.html [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+        <h2>Experience</h2>
+        <ul>
+            <li>Four years of learning Bachelor Of Engineering in Anna Unversity</li>
+        </ul>
+    </section>
+    <section id="Connect">
+        <h2>Connect</h2>
+        <ul>
+            <li><img src="/Users/umasekar/Documents/Uma/CodeTheDream/haumea-repos/uma-venki-haumea/Images/5296501_linkedin_network_linkedin logo_icon.svg"><a href="https://www.linkedin.com/in/uma-sekar-4073982a4">Linkedln</a></li>
+            <li><img src="/Users/umasekar/Documents/Uma/CodeTheDream/haumea-repos/uma-venki-haumea/Images/211904_social_github_icon.svg"><a href="https://github.com/umavenki/uma-venki-haumea">Github</a></li>
```
Comment (toma63) [inline index.html:31]: These src paths will only work on your machine.  You should use a relative path './Images/...'.

**Review by toma63 — APPROVED** [review]
This now meets requirements.  The icon images need to be corrected to use relative paths.  You can do this in the next lesson submission since they are optional.

**Pull request merged**